src/domain/ruleset-repository.spec.ts

```diff
@@ -151,6 +151,15 @@ describe("create", () => {
         email: "json@bearded.ca",
       });
     });
+
+    test("does not complain when the config file is empty", async () => {
+      mockRulesetFiles({
```

Comment:
instead of adding properties to the mock, can you just use an empty string? Seems a bit overengineered

Reply:
I could pass a `content` property that's just the raw string.

Reply:
Added.